src/components/bridge/BridgeSelection.vue

```diff
@@ -241,6 +244,10 @@ export default defineComponent({
       return isH160.value && (isAstar.value || isAstarZkEvm.value);
     });
 
+    const isEnableCelerBridge = computed<boolean>(() => {
```

Comment:
You can make this statemen shoreter

` const isEnableCelerBridge = computed<boolean>(() => false);`

Reply:
Yes, thank you! fixed in [8b0790b](https://github.com/AstarNetwork/astar-apps/pull/1353/commits/8b0790b0367905c02aaa8e5497dd54531f4bd9b9) commit